examples/examplesVersionDependency.ttl

```diff
@@ -26,10 +36,27 @@
 	] ;
 	.
 
+<https://ontologies.exco.com/unversionedOntology>
+	a owl:Ontology ;
+	skos:scopeNote "This ontology does not declare a version."^^xsd:string ;
+	skos:prefLabel "Unversioned Ontology"^^xsd:string ;
+	.
+
+<https://ontologies.exco.com/unconventionalVersioning>
```

Comment:
```suggestion
<https://ontologies.exco.com/nonStandardVersioning>
```

"unconventional" means something slightly different..."non-conventional" might also be OK. Or could be "nonSemantic".